ui/components/app/confirm/info/row/address.test.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { CHAIN_IDS } from '../../../../../../shared/constants/network';
+import mockState from '../../../../../../test/data/mock-state.json';
+import { renderWithProvider } from '../../../../../../test/lib/render-helpers';
+import { shortenAddress } from '../../../../../helpers/utils/util';
+import configureStore from '../../../../../store/store';
+import { ConfirmInfoRowAddress } from './address';
+import { TEST_ADDRESS } from './constants';
+
+const render = (
+  storeOverrides: Record<string, any> = {},
+  address: string = TEST_ADDRESS,
+) => {
+  const store = configureStore({
+    metamask: { ...mockState.metamask },
+    ...storeOverrides,
+  });
+
+  return renderWithProvider(<ConfirmInfoRowAddress address={address} />, store);
+};
+
+describe('ConfirmInfoRowAddress', () => {
+  it('should render appropriately with PetNames enabled', () => {
+    const { container } = render({
+      metamask: {
+        ...mockState.metamask,
+        preferences: {
+          ...mockState.metamask.preferences,
+          petnamesEnabled: true,
+        },
+      },
+    });
+
+    expect(container).toMatchSnapshot();
+  });
+
+  describe('should render appropriately with PetNames disabled', () => {
```

Comment:
More suggestions here on how describe/it may be used

https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#use-describe-to-group-tests-for-the-same-functionmethod

Reply:
Hey @segun, thanks for your review. I didn't get your point on this comment.

I grouped test cases with PetNames disabled under the same `describe()` in a way that matches [this section of the documentation you link](https://github.com/MetaMask/contributor-docs/blob/main/docs/unit-testing.md#-use-describe-to-group-tests-under-scenarios).

Can you clarify what specific change you were looking for?